tests/integration/helm/util.go

```diff
@@ -93,6 +93,30 @@ func InstallIstio(t test.Failer, cs cluster.Cluster,
 	}
 }
 
+// InstallIstioWithRevision install Istio using Helm charts with the provided
+// override values file and fails the tests on any failures.
+func InstallIstioWithRevision(t test.Failer, cs cluster.Cluster,
+	h *helm.Helm, fileSuffix, revision, overrideValuesFile string) {
+	CreateNamespace(t, cs, IstioNamespace)
+
+	// Upgrade base chart
```

Comment:
cc @Monkeyanator this is how users use the base chart in Helm based installs. 

cc @nrjpoddar This isn't exactly precise as CRDs aren't upgrade, but I'm hoping this alleviates some of the concern around https://github.com/istio/istio/pull/31790#issuecomment-816668269. Anything else I should add in testing?